packages/block-directory/src/components/downloadable-block-info/index.js

```diff
@@ -16,7 +16,9 @@ function DownloadableBlockInfo( { description, activeInstalls, humanizedUpdated
 					<Icon icon="chart-line"></Icon>{ sprintf( _n( '%d active installation', '%d active installations', activeInstalls ), activeInstalls ) }
 				</div>
 				<div className="block-directory-downloadable-block-info__column">
-					<Icon icon="update"></Icon><span aria-label={ sprintf( __( 'Updated %s' ), humanizedUpdated ) }>{ humanizedUpdated }</span>
+					<span aria-label={ sprintf( __( 'Updated %s' ), humanizedUpdated ) }>
```

Comment:
Without an icon I don't think this `span` with `aria-label` makes too much sense anymore. A11y feedback would be good here.

Reply:
Definitely agreed, otherwise the aria-label will override the actual text. And since the actual text is even more descriptive now, that would mean a loss of information. So yes, the span should go away including the aria-label. The strong with info text is totally sufficient here.

Reply:
Yeah totally. Thanks for the heads up!

Reply:
In a parallel conversation, I was made aware of these prototypes:
https://www.figma.com/file/QKhoOKXkBN2mHacqvrtdNuI6/Gutenberg-Block-Library-Installation?node-id=370%3A0

It appears like the current implementation was missing the "Updated" string but maintained the copy. The design could accommodate it by having a smaller font size. With that in mind. I have updated this PR to be:

![alt text](https://d.pr/i/yCPWqa.png "Newly Updated")

Should this be tagged with "Needs Design Feedback" in case there have been other conversations had here?